packages/core/src/validate/run.ts

```diff
@@ -111,11 +177,13 @@ export function validate(solcOutput: SolcOutput, decodeSrc: SrcDecoder): Validat
 function* getConstructorErrors(contractDef: ContractDefinition, decodeSrc: SrcDecoder): Generator<ValidationError> {
   for (const fnDef of findAll('FunctionDefinition', contractDef)) {
     if (fnDef.kind === 'constructor' && ((fnDef.body?.statements.length ?? 0) > 0 || fnDef.modifiers.length > 0)) {
-      yield {
-        kind: 'constructor',
-        contract: contractDef.name,
-        src: decodeSrc(fnDef),
-      };
+      if (!skipCheck('constructor', contractDef) && !skipCheck('constructor', fnDef)) {
```

Comment:
Was there a reason why you used the `prune` argument to `findAll` in some cases and not others?

Reply:
MY bad, I should have used prune here.